src/gui/exportOptions.cpp

```diff
@@ -479,6 +479,60 @@ void FurnaceGUI::drawExportDMF(bool onWindow) {
   }
 }
 
+void FurnaceGUI::drawExportMML(bool onWindow) {
+  exitDisabledTimer = 1;
+
+  ImGui::Text(_("export in various Music Macro Language formats."));
+
+  // Dropdown for MML type
+  static int mmlType = 4; // default to mmlgb
+  const char* mmlTypeNames[] = {
+    "[NES] PPMCK (unimplemented)",
+    "[PC-98] FMP-compatible (unimplemented)",
+    "[PC-98] PMD-style (unimplemented)",
+    "[SNES] AddMusicK (unimplemented)",
+    "[Gameboy] mmlgb"
+  };
+
+  ImGui::Text(_("MML Type:"));
+  ImGui::Combo("##mmlType", &mmlType, mmlTypeNames, IM_ARRAYSIZE(mmlTypeNames));
```

Comment:
Don't expose for now - once we have another type implemented we can show this combo box.